php-checks/src/main/java/org/sonar/php/checks/HardCodedCredentialsInFunctionCallsCheck.java

```diff
@@ -175,12 +175,12 @@ static class JsonSensitiveFunctionsReader {
     private JsonSensitiveFunctionsReader() {
     }
 
-    private static Map<String, SensitiveMethod> parseSensitiveFunctions() {
+    static Map<String, SensitiveMethod> parseSensitiveFunctions(String location, Set<String> fileNames) {
       Map<String, SensitiveMethod> sensitiveFunctions = new HashMap<>();
 
-      for (String json_location : SENSITIVE_FUNCTIONS_JSON) {
+      for (String json_location : fileNames) {
```

Comment:
Nitpick: rename to `jsonLocation`, or even change to `filename`?

Reply:
Good catch!